core/plugins/org.fusesource.ide.help/contexts.xml

```diff
@@ -362,10 +362,6 @@
         <description>Azure Storage Queue Service Component Options</description>
         <topic href="azure-queue-component.html" label="Azure Storage Queue Service"/>
     </context>
-    <context id="beanEIP">
```

Comment:
We should not update the help plugin for now. This will be part of @MelissaFlinn PR whenever it is ready. Obviously you can do it locally for testing purposes though.